.github/workflows/unit-tests.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         with:
           path: ~\AppData\Local\pip\Cache
           key: ${{inputs.os}}-python-${{inputs.python-version}}
+      - name: Add MSBuild to PATH
+        if: inputs.os == 'windows-latest'
+        uses: microsoft/setup-msbuild@v1
       - name: Install dependencies
         run: |
             make install-test-requirements
```

Comment:
I am surprise you don't need to add 3.11 here. Does that means test can be run in parallel for 3.11? Or is it because the `moto` version is now upgrade so we don't need to handle 3.10 specially anymore?

Reply:
The new `moto` version now allows for parallel test execution for Python 3.10. I believe before when we [added 3.10 support](https://github.com/kedro-org/kedro/pull/1275#:~:text=moto%20had%20to,run%20in%20parallel.) `moto` newer version broke parallel tests for 3.10 hence it was made sequential but I believe with the newer `moto` version we have pinned this has resolved the issue.

Reply:
That's amazing! 🤩

Reply:
Wow! Good that I asked. This is great, I have look into the s3fs repository and basically they have to use the server mode for `moto` to run tests. The newer version of `moto` seems supporting this better and potentially we can unblock this ticket.

- https://github.com/kedro-org/kedro/issues/1995